### ISSUE TITLE
Identify duplicate selectors

### ISSUE THREAD

**Issue by Pikamander2** [CSSLint/csslint#777]
Despite being 100% redundant, this rule doesn't generate any warnings/errors:

```
div, div
{
    display: none;
}
```
